dll/win32/shell32/dialogs/folder_options.cpp

```diff
@@ -1987,16 +2204,24 @@ FolderOptionsFileTypesDlg(
 
         case WM_NOTIFY:
             lppl = (LPNMLISTVIEW) lParam;
-
+            if (lppl->hdr.code == LVN_DELETEALLITEMS)
+            {
+                return FALSE;   // send LVN_DELETEITEM
+            }
+            if (lppl->hdr.code == LVN_DELETEITEM)
```

Comment:
Not using "else if" or a "switch (lppl->hdr.code)" ?

Reply:
OK. I used switch.